docs/syntax.md

```diff
@@ -56,7 +56,8 @@ These options make up the basic actions for cluster creation and maintenance.
 | `--start`  | Reactivate an offline cluster. Read [Start a Cluster](using_vfxt_py.md#start-a-cluster) for details. |
 | `--add-nodes` | Add nodes to a cluster. Read [Add Nodes to a Cluster](using_vxt_py.md#add-nodes-to-a-cluster) for details. |
 | `--upgrade` | Updates the Avere OS software for the cluster. Used with `--upgrade-url` and `--upgrade-non-ha` (described below). Read [Update Software](using_vfxt_py.md#update-software) for more information. |
-| `--telemetry` | Starts a support data upload. You must accept privacy terms from the Avere Control Panel in order to use this option. Read [Using the Avere Control Panel Support Tab](<http://library.averesystems.com/ops_guide/4_7/support_overview.html>) for details. |
+| `--telemetry` | Starts a support data upload. You must accept privacy terms from the Avere Control Panel in order to use this option. Read [Using the Avere Control Panel Support Tab](<https://azure.github.io/Avere/legacy/ops_guide/4_7/html/support_overview.html>) for details. |
+| `--telemetry-mode` *mode* | **XXX ??? XXX** |
```

Comment:
Refer to the xmlrpc developers guide regarding support.listNormalModes for specific options.